_includes/risk-indicator-table.md

```diff
@@ -1,4 +1,4 @@
-|          | └➔&nbsp;`riskIndicator`               | `array`      | This **optional** object consist of information that helps verifying the payer. Providing these fields decreases the likelyhood of having to promt for 3-D Secure authenticaiton of the payer when they are authenticating the purchacse.                                                                                                                                                                                                                                                                                                                         |
+|          | └➔&nbsp;`riskIndicator`               | `array`      | This **optional** object consist of information that helps verifying the payer. Providing these fields decreases the likelyhood of having to promt for {% if documentation_section== "card" %} [3-D Secure authenticaiton](/payments/{{ documentation_section }}/other-features#3-d-secure-20)  {% else %} [3-D Secure authenticaiton](/{{ documentation_section }}/other-features#3-d-secure-20) {% endif %} of the payer when they are authenticating the purchase.                                                                                                                                                                                                                                                                                                                                                                                                                           |
```

Comment:
Declare (`capture`) the `3ds2link` somewhere at the top of the file so you don't need `if` statements inside the table, like so:

```liquid
{%- capture 3ds2link -%}
    {%- if documentation_section== "card" -%}
        /payments/{{ documentation_section }}/other-features#3-d-secure-2
    {%- else -%}
        /{{ documentation_section }}/other-features#3-d-secure-2
    {%- endif -%}
{%- endcapture -%}
```

```suggestion
|          | └➔&nbsp;`riskIndicator`               | `array`      | This **optional** object consist of information that helps verifying the payer. Providing these fields decreases the likelihood of having to promt for [3-D Secure authentication]({{ 3ds2link }}) of the payer when they are authenticating the purchase.                                                                                                                                                                                                                                                                                                                                                                                                                           |
```